### PR TITLE
Completed javascript-functions exercise.

### PR DESCRIPTION
**Pull request by arjunkahlon**
Screen Shot 2022-07-06 at 3 52 39 PM (2)

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Function Definitions
+
+function convertMinutesToSeconds(minutes) {
+  return minutes * 60;
+}
+
+function greet(name) {
+  console.log('Hey,', name);
```
Comment (codingcodymiller) [inline javascript-functions/main.js:8]: You should be `return`ing the greeting

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by arjunkahlon** [comment]
Screen Shot 2022-07-06 at 3 56 15 PM (2)

**Comment by arjunkahlon** [comment]
Screen Shot 2022-07-06 at 3 57 29 PM (2)

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Submitting another review, my previous feedback has not been addressed

**Comment by arjunkahlon** [comment]
Screen Shot 2022-07-06 at 4 01 29 PM (2)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**